backend/src/database/notifications/notifications.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
+import { NotificationsDBService } from './notifications.service';
+import { NotificationsProviders } from './notifications.providers';
+import { DatabaseModule } from '../database.module';
+
+@Module({
+    imports: [DatabaseModule],
```

Comment:
Ich habe auf meinem aktuellen PR die Database Module etwas angepasst (auf den "more Nest.js" Weg). Wäre cool wenn du die Anpassungen auch hier übernehmen würdest.  Die Änderungen siehst du eigentlich ganz gut in meinem PR. Wenn du Hilfe beim anpassen brauchst schreib mich einfach an.

Reply:
Schaue ich mir an 👍